src/flexbox/nasa.css

```diff
@@ -0,0 +1,72 @@
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
```

Comment:
since your two elements are in a column and you've set the size of each element to make up 100%, `justify-content` isn't necessary.